src/tokens/Assets.tsx

```diff
@@ -249,12 +265,14 @@ function AssetsScreen({ navigation, route }: Props) {
       ? [{ data: tokenItems }]
       : activeTab === AssetTabType.Positions
       ? positionSections
+      : nfts.length
+      ? [{ data: nfts }]
```

Comment:
nit: might be worth just doing an if else or switch and avoid the nested ternary.